paw-note.el

```diff
@@ -654,8 +654,7 @@ Bound to \\<C-cC-c> in `paw-note-mode'."
                   (if (s-blank-str? note)
                       (overlay-put ov 'after-string nil)
                     (overlay-put ov 'after-string nil)
-                    (overlay-put ov 'after-string paw-comment-button))
-                  (save-excursion (paw-add-inline-annotation ov)))))))
```

Comment:
Oh, maybe you need to pull updates first, I just added the inline-annotation feature.